socorro/unittest/signature/test_rules.py

```diff
@@ -972,9 +965,7 @@ def test_action_1(self):
 
         raw_crash = {
             'JavaStackTrace': (
-                '   SomeJavaException: %s  \n'
-                'at org.mozilla.lars.myInvention('
-                'larsFile.java)' % ('t' * 1000)
+                '   SomeJavaException: t  \nat org.mozilla.lars.myInvention(larsFile.java)'
```

Comment:
I messed up--I need to re-fix this because it should be testing that the description gets dropped if it's too long.